curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e78a7ea4a168de4e6a38.md

```diff
@@ -30,7 +30,7 @@ assert.match(code, /openTaskFormBtn\.addEventListener\(('|"|`)click\1/)
 Your event listener should include a callback function for the second argument using arrow syntax without curly braces. Then you should use the `classList` property and its `toggle` method on the `taskForm`, to toggle the class `hidden`. 
 
 ```js
-assert.match(code, /openTaskFormBtn\.addEventListener\(('|"|`)click\1\,\s*\(\)\s*\s*=>\s*taskForm\.classList\.toggle\(\1hidden\1\)\s*\);?/)
+assert.match(code, /openTaskFormBtn\.addEventListener\(\s*("|')click\1\s*,\s*\(\)\s*=>\s*taskForm\.classList\.toggle\(\s*("|')hidden\2\s*\)/)
```

Comment:
> Re: backtick, I think the backtick should be added back to the regex group, not because it should be allowed here, but removing it from this specific test would cause an inconsistency within the test suite (the test above this allows backtick).

I agree with you.

Are you suggesting that this PR should only add the `\s*` and let #52995 handle the quotation mark check?